packages/main/src/plugin/tasks/task-manager.ts

```diff
@@ -62,6 +62,11 @@ export class TaskManager {
             type: 'boolean',
             default: false,
           },
+          [`${ExperimentalTasksSettings.SectionName}.${ExperimentalTasksSettings.Toast}`]: {
+            description: 'Display a notification toast when task is created',
+            type: 'boolean',
+            default: false,
+          },
```

Comment:
not hidden ?

Reply:
it seems that 'description: 'Show running tasks in the status bar',' is not hidden as well

so either both of them need to be hidden or all of them need to be visible

Reply:
Oh yeah I thought it was hidden, weird